socorro/external/es/crashstorage.py

```diff
@@ -53,6 +57,61 @@ def is_valid_key(key):
     return bool(VALID_KEY.match(key))
 
 
+def remove_bad_keys(data):
+    """Removes keys from the top-level of the dict that are bad
+
+    Good keys satisfy the following properties:
+
+    * have one or more characters
+    * are composed of a-zA-Z0-9_-
+
+    Anything else is a bad key and needs to be removed.
+
+    Note: This modifies the data dict in-place and only looks at the top level.
+
+    :arg dict data: the data to remove bad keys from
+
+    """
+    if not data:
+        return
+
+    # Copy the list of things we're iterating over because we're mutating
+    # the dict in place.
+    for key in list(data.keys()):
+        if not is_valid_key(key):
+            del data[key]
+
+
+def truncate_keyword_field_values(fields, data):
+    """Truncates values of keyword fields greater than 10,000 characters
```

Comment:
Maybe just reference the constant name here so we don't have to change it in two places in the future?

Reply:
Good idea!